applications/FluidDynamicsApplication/custom_utilities/two_fluid_navier_stokes_data.h

```diff
@@ -345,6 +352,18 @@ void CalculateEffectiveViscosityAtGaussPoint()
     }
     else this->EffectiveViscosity = DynamicViscosity;
 }
+
+void ComputeDarcyTerm()
+{
+    array_1d<double, 3> advec_velocity(3, 0.0);
+    for (size_t i = 0; i < TNumNodes; i++) {
+        for (size_t j = 0; j < TDim; j++) {
+            advec_velocity[j] += this->N[i] * (Velocity(i, j) - MeshVelocity(i, j));
```

Comment:
We always denote this term as convective velocity. It would be nice if you change it to match the other elements.